src/arithmetic/string_funcs/string_funcs.c

```diff
@@ -412,11 +418,17 @@ void Register_StringFuncs() {
 	AR_RegFunc(func_desc);
 
 	types = array_new(SIType, 1);
-	array_append(types, SI_ALL);
+	array_append(types, STRINGABLE | T_NULL);
```

Comment:
why this change?

Reply:
I think this is the correct change to be made
From Neo4j's doc:
```
This function will return an error if provided with an expression that is not an integer, float, string, boolean, point, duration, date, time, localtime, localdatetime or datetime value.
```